deploy/olm-catalog/container-jfr-operator-bundle/0.4.0/container-jfr-operator-bundle.v0.4.0.clusterserviceversion.yaml

```diff
@@ -38,7 +38,7 @@ metadata:
             "archive": true,
             "duration": "30s",
             "eventOptions": [
-              "ALL"
+              "template=ALL"
```

Comment:
Perhaps we should leave the 0.4.0 CSV as-is, and do a new "release" with version bump and updated CSV?

Reply:
Sure, I'll back out that part of the change. I think you did the last CSV update - do you want to do this next one too?

Reply:
Sure, I can do that.